backend/src/repository/dalBoard.ts

```diff
@@ -47,12 +56,36 @@ export const update = async (id: string, board: Partial<BoardModel>) => {
   }
 };
 
+export const remove = async (id: string) => {
+  const session = await mongoose.startSession();
+  session.startTransaction();
+  try {
+    const deletedBoard = await Board.findOneAndDelete({ boardID: id });
+    if (deletedBoard) {
+      await dalPost.removeByBoard(id);
+      await dalComment.removeByBoard(id);
```

Comment:
I think you need to add dalVote.removeByBoard here as well to remove all the upvotes for this board.

Reply:
oh ye i forgot to push the final changes xd. ill do it rn.